README.md

```diff
@@ -93,6 +93,21 @@ if err != nil {
 }
 ```
 
+`Publish` will block until either the publish is acknowledged or failed to
+deliver.
+
+Alternatively you can use `PublishAsync` which does not block:
```

Comment:
Alternatively, you can use `PublishAsync, which doesn't wait for `publish` to finish.